src/docfx/lib/XrefResolvingResult.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Docs.Build;
+
+internal readonly struct XrefResolvingResult
```

Comment:
This structure is actually representing a link:
```suggestion
internal readonly struct XrefLink
```

Reply:
Also make `XrefResolver.ResolveXrefByHref` method return `(Error, XrefLink)` to make this type propagate fullstack.

Reply:
Done